rpa-home/modules/ROOT/pages/invoke-rpa-from-flow-build.adoc

```diff
@@ -0,0 +1,47 @@
+
+# Build an Automation
+
+Implement your automation in RPA Builder.
+
+## Before You Begin
```

Comment:
This heading is not needed.

Reply:
Replaced wording:

Reply:
* For input and output parameters of a process, only use variable types that are supported in Salesforce Invocable Actions as RPA Builder Activity Parameters:
+
The following table shows possible mappings: